### PR TITLE
[V4] homogenized styles for labels and icons in wallet details page

### PR DESCRIPTION
**Pull request by JDonadio**

### CODE REVIEW

File: src/pages/wallet-details/wallet-details.scss [hunk 1]
```diff
@@ -69,49 +69,44 @@ page-wallet-details {
...
-      margin: 0 3px;
-      padding-top: 5px;
-    }
     ion-spinner {
       float: right;
       width: 20px;
       height: 20px;
-      margin: 5px 5px 0 0;
-      * {
+      &.spinner-crescent circle {
```
Comment (cmgustavo) [inline src/pages/wallet-details/wallet-details.scss]: `spinner-crescent` is for Android only. Using `*` will work on iOS also.

Comment (JDonadio) [reply src/pages/wallet-details/wallet-details.scss]: Done. Thanks!

**Comment by JDonadio** [comment]
Updated

**Comment by gabrielbazan7** [comment]
ACK

**Pull request merged**